app/src/editor/shared/components/modules/Scheduler.jsx

```diff
@@ -0,0 +1,714 @@
+import React, { Fragment } from 'react'
+import uuid from 'uuid'
+import { arrayMove } from 'react-sortable-hoc'
+
+import ModuleSubscription from '../ModuleSubscription'
+import SortableList from '$shared/components/SortableList'
+import { withHover } from '$shared/components/WithHover'
+import SvgIcon from '$shared/components/SvgIcon'
+import TextInput from '$editor/shared/components/TextInput'
+
+import styles from './Scheduler.pcss'
+
+const schedulerOptions = {
+    '0': 'hour',
+    '1': 'day',
+    '2': 'week',
+    '3': 'month',
+    '4': 'year',
+}
+
+const formatDay = (d) => {
+    let ending = 'th'
+
+    if (d === 1 || (d >= 20 && d % 10 === 1)) {
+        ending = 'st'
+    } else if (d === 2 || (d >= 20 && d % 10 === 2)) {
+        ending = 'nd'
+    } else if (d === 3 || (d >= 20 && d % 10 === 3)) {
+        ending = 'rd'
+    }
+
+    return `${d}${ending}`
+}
+const minutes = Array.from(Array(60), (d, i) => i)
+const hours = Array.from(Array(24), (d, i) => i)
+const weekdays = {
+    '2': 'Monday',
+    '3': 'Tuesday',
+    '4': 'Wednesday',
+    '5': 'Thursday',
+    '6': 'Friday',
+    '7': 'Saturday',
+    '1': 'Sunday',
+}
+const days = Array.from(Array(31), (d, i) => i + 1).reduce((result, d) => ({
+    ...result,
+    [d]: formatDay(d),
+}), {})
+const months = {
+    '1': 'January',
+    '2': 'February',
+    '3': 'March',
+    '4': 'April',
+    '5': 'May',
+    '6': 'June',
+    '7': 'July',
+    '8': 'August',
+    '9': 'September',
+    '10': 'October',
+    '11': 'November',
+    '12': 'December',
```

Comment:
I18n for this easy or no?

Reply:
Not sure if we can get them automatically from the `I18n` lib but at least we can translate it ourselves. Just need to move them to be generated at the component level where we have access to `I18n` object.